sdks/python/apache_beam/runners/interactive/pipeline_fragment.py

```diff
@@ -23,6 +23,7 @@
 from apache_beam.pipeline import AppliedPTransform
 from apache_beam.pipeline import PipelineVisitor
 from apache_beam.runners.interactive import interactive_environment as ie
+from apache_beam.runners.interactive import interactive_runner as ir
```

Comment:
Let's move the import down to where it's used and only import the class. Keeping it here might cause import loops.

Reply:
Done. Thanks.